src/events/cd-booking-create-account.vue

```diff
@@ -113,61 +117,53 @@
         this.formValidated = true;
         return (!this.errors.any() && !!this.recaptchaResponse);
       },
-      submitAccount() {
-        return this.register()
-          .then(this.addChildren)
-          .then(this.joinDojo)
-          .then(this.bookTickets);
+      async submitAccount() {
+        await this.register();
+        await this.addChildren();
+        await this.joinDojo();
+        return this.bookTickets();
       },
-      register() {
+      async register() {
```

Comment:
Technically not, but it does return a promise, so I personally like having async just for readability. At a glance, you know it returns a promise.